src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import {
+  createStore, combineReducers, applyMiddleware,
+} from 'redux';
+import logger from 'redux-logger';
+
+import bookReducer from './books/books';
```

Comment:
- [ ] Kindly this reducer comes from `'../redux/books/Books'` Books not books, to solve these errors.

![image](https://user-images.githubusercontent.com/35535130/163198764-3849013c-a5b8-4a63-b2ac-af19736e4ee0.png)